src/item.rs

```diff
@@ -34,6 +35,7 @@ impl<'a> Item<'a> {
         let item_proxy = ItemProxyBlocking::builder(&conn)
             .destination(SS_DBUS_NAME)?
             .path(item_path.clone())?
+            .cache_properties(CacheProperties::No)
```

Comment:
With 2.0.1 this shouldn't be needed as properties are cached lazily by default (as they were supposed to).

Reply:
Some tests still fail w/o disabling caching like https://github.com/hwchen/secret-service-rs/blob/b44a455626873283dfd52c9a099c69b4742d14ec/src/item.rs#L175. The `Item`'s label can still be read after deleting it and there's no call to `org.freedesktop.DBus.Properties.Get` before getting the label again.

Reply:
Interesting, this seems like a zbus bug then. I'll investigate soon but I good that you've a simple good workaround for now.